spec/models/timed_transitions/transitioner_spec.rb

```diff
@@ -106,7 +108,7 @@
               end
             end
 
-            it 'calls archive if last state change more than 16 weeks ago' do
+            it 'amends claim state to archives pending review' do
```

Comment:
Non-blocking minor typo, should it read 'amends claim state to archived pending review'

Reply:
cheers, will fix in my next pr on top - test suite takes ages 😞